packages/arb-token-bridge-ui/src/hooks/useNetworksAndSigners.tsx

```diff
@@ -151,7 +151,10 @@ function getProviderName(provider: any): ProviderName | null {
 }
 
 async function isBlocked(address: string): Promise<boolean> {
-  if (process.env.NODE_ENV !== 'production') {
+  if (
+    process.env.NODE_ENV !== 'production' ||
```

Comment:
nit: Maybe we can refactor and reuse this little logic from `CommonUtils.ts` - something like `isLocalOrE2EEnv()`?
This is also being used in `_app.tsx`.

Reply:
done

Reply:
Had to revert, couldn't read env vars from utils